src/domain.test.ts

```diff
@@ -49,7 +49,15 @@ describe("toggle", () => {
       configuration.previewMarkdownFiles
     );
   });
+  it("should override default slide settings with slidesrc configuration", async () => {
```

Comment:
👍 Nice, thanks for adding tests with your PR. 

That really helps me review (and that's good!).